src/graph_notebook/magics/graph_magic.py

```diff
@@ -739,7 +741,7 @@ def load(self, line='', local_ns: dict = None):
             value=args.loader_arn,
             placeholder='Type something',
             disabled=False,
-            layout = widgets.Layout(width=widget_width)
+            layout=widgets.Layout(width=widget_width)
```

Comment:
This should have a space on either side of the =

Reply:
I changed this due to a failed styling check; the PEP8 styling guide states that there should not be any spaces around `=` in function keyword arguments.

https://www.flake8rules.com/rules/E251.html

Reply:
TIL - good to know